apis/dataset_metadata/dataset_related_identifier.py

```diff
@@ -111,16 +115,18 @@ class DatasetRelatedIdentifierUpdate(Resource):
     @api.response(204, "Success")
     @api.response(400, "Validation Error")
     def delete(
-            self,
-            study_id: int,
-            dataset_id: int,  # pylint: disable= unused-argument
-            related_identifier_id: int,
+        self,
+        study_id: int,
+        dataset_id: int,  # pylint: disable= unused-argument
```

Comment:
Why does this have the pylint statement?

Reply:
here study_id is not used inside endpoint but its called in the url so that has to remain there. thats why pylint complains why we dont use it in the endpoint

Reply:
Shouldn't study_id have the unused argument statement then?